### PR TITLE
add dumping essential.exefs to finalize

### PR DESCRIPTION
**Pull request by lifehackerhansol**
**Description**

By request of @lilyuwuu `essential.exefs` is now dumped during Finalizing Setup. `.noexefs` is no more.

GM9Megascript has everything.

### CODE REVIEW

**Comment by lifehackerhansol** [comment]
If we do end up going the long way for that, you may as well dump boot9 from MEMORY VIRTUAL as well. It wouldn't make sense to visit "Dump options" once to do that.

I guess I'll revert that commit here.

File: _pages/en_US/finalizing-setup.txt [hunk 1]
```diff
@@ -139,6 +139,11 @@ If, before following this guide, you already had an EmuNAND setup and would like
 1. Press (B) to return to the main menu
 1. Select "Exit"
 1. Press (A) to relock write permissions if prompted
+1. Navigate to `[S:] SYSNAND VIRTUAL`
+1. Press (A) on `essential.exefs` to select it
+1. Select "Copy to 0:/gm9/out"
+  + If prompted that "Destination already exists", press (A) on "Overwrite file(s)"
```
Comment (lilyuwuu) [inline _pages/en_US/finalizing-setup.txt:145]: Maybe use `If you see "Destination already exists"`

**Review by lilyuwuu — APPROVED** [review]
lgtm

File: _pages/en_US/finalizing-setup.txt (hunk 1 above)
Comment (lifehackerhansol) [reply _pages/en_US/finalizing-setup.txt]: Amended.

**Pull request merged**